modules/python/init.zsh

```diff
@@ -93,7 +93,9 @@ if (( $+VIRTUALENVWRAPPER_VIRTUALENV || $+commands[virtualenv] )) && \
   VIRTUAL_ENV_DISABLE_PROMPT=1
 
   # Enable 'virtualenv' with 'pyenv'.
-  if (( $+commands[pyenv] && $+commands[pyenv-virtualenv-init] )); then
+  if (( $+commands[pyenv] )) && \
+     pyenv commands | command grep -q virtualenv-init
```

Comment:
This should probably be a check for `[[ -n $(pyenv commands | command grep -q virtualenv-init) ]]` (https://google.github.io/styleguide/shell.xml?showone=Testing_Strings#Testing_Strings) and the `then` should be on the same line as the conditional (https://google.github.io/styleguide/shell.xml?showone=Loops#Loops)